File: C#/1_two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int x = 0; x < nums.Length; x++)
        {
            for (int z = x + 1; z < nums.Length; z++)
            {
                if (nums[x] == target - nums[z])
                {
                    return new int[] {x, z};
                }
            }
        }
        return new int[] {0, 0};
    }
}

File: Python/1_two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for first_index, first_num in enumerate(nums):
            for second_index in range(first_index + 1, len(nums)):
                if first_num + nums[second_index] == target:
                    return [first_index, second_index]